src/utils/paymentPlanStrings.tsx

```diff
@@ -23,6 +23,19 @@ export const paymentPlanShorthandName = (payment: EligibilityPlan): ReactNode =>
   }
 }
 
+const withNoFee = (payment: EligibilityPlanToDisplay) => {
+  if (
+    payment.payment_plan?.every((plan) => plan.customer_fee === 0 && plan.customer_interest === 0)
+  ) {
+    return (
+      <>
+        {' '}
```

Comment:
I think it's adding a space when we add this wording. But we don't want this space to be part of the translation to avoid mistakes.
We could replace it by a `&nbsp;` but it does not change anything

Reply:
FWIW i'd rather have {' '}